movies/webpack.config.prod.js

```diff
@@ -0,0 +1,88 @@
+
```

Comment:
consider creating `webpack.config.common` and using https://github.com/survivejs/webpack-merge for better reuse

Reply:
And please, commint your package-lock

Reply:
Package-lock  and webpack-merge has been added.